main.tf

```diff
@@ -1,21 +1,30 @@
 locals {
-  access_token_json = var.use_ssm_parameter_store ? (
-    {
-      "secrets" : [{
-        "name" : "LaceworkAccessToken",
-        "valueFrom" : local.ssm_parameter_arn
-      }]
-    }
+  secrets_json = var.use_ssm_parameter_store ? ({
+    "secrets" : [
+      { "name" : "LaceworkAccessToken", "valueFrom" : local.ssm_parameter_arn }
+    ]
+  }) : ({})
+
+  environment_json = (!var.use_ssm_parameter_store) ? (
+    length(var.lacework_server_url) > 0 ? ({
```

Comment:
Could this conditional logic be removed if the server url is default is set to  "https://api.lacework.net" ?